packages/rollup-plugin-html/src/output/emitAssets.ts

```diff
@@ -9,6 +11,29 @@ export interface EmittedAssets {
   hashed: Map<string, string>;
 }
 
+const allowedFileExtensions = [
+  // https://www.w3.org/TR/html4/types.html#:~:text=ID%20and%20NAME%20tokens%20must,tokens%20defined%20by%20other%20attributes.
+  /.*\.svg(#[A-Za-z][A-Za-z0-9\-_:.]*)?/,
+  /.*\.png/,
+  /.*\.jpg/,
+  /.*\.jpeg/,
+  /.*\.webp/,
+  /.*\.gif/,
+  /.*\.avif/,
+  /.*\.woff2/,
+  /.*\.woff/,
+];
```

Comment:
why do you want to use the allowed extensions list? are there some extensions that should not be handled?

Reply:
We're not really sure what people may add in `url`, so I figured be on the safe side

Reply:
with current approach we'll need to maintain a list of extensions and update when new types appear or if we forgot to include smth

instead I'd rather prefer to have a `blocklist` and make it configurable, we can do that later too